docs/pull-request-faq.md

```diff
@@ -0,0 +1,37 @@
+# Yeti Pull Request FAQ
+
+This document addresses common questions that come up when you are ready to create a pull request and submit your work.
+
+## What are the PR template sections for?
+
+**Ticket**
+
+- Provide a link to the work ticket so that the reviewer can easily see the task requirements.
```

Comment:
Small nitpick here but I think we might want to say `relevant ticket` instead of `work ticket` 😄